.github/workflows/fuzz.yml

```diff
@@ -1,6 +1,11 @@
 name: Fuzz
 
-on: [push, pull_request]
+on:
+  push:
+    branches:
+      - master
```

Comment:
We might also want 0.29.x (and other version branches), what do you think @apoelstra?

Reply:
On those branches, can we run regular CI **except** the 1.41 tests, because those typically break soon after release, and it's annoying to backport `cargo update` commands. Similarly we'd want to disable clippy and fmt, which drift over time.

I think this'll be a bit more complicated, so I'll move to merge this PR and we can do that stuff in a later one.

Reply:
I also think that such a change wouldn't actually take effect unless we backport it to the release branches.

Reply:
Yes, that is correct. We would need to backport to any branches that want these changes applied.